lib/ui/views/ib/builders/ib_pop_quiz_builder.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:markdown/markdown.dart' as md;
+import 'package:mobile_app/ui/views/ib/components/ib_pop_quiz.dart';
+import 'package:mobile_app/viewmodels/ib/ib_page_viewmodel.dart';
+
+class IbPopQuizBuilder extends MarkdownElementBuilder {
+  final BuildContext context;
+  final IbPageViewModel model;
+
+  IbPopQuizBuilder({this.context, this.model});
+
+  @override
+  Widget visitElementAfter(md.Element element, TextStyle preferredStyle) {
+    var _rawContent = element.textContent;
+    var _popQuizQuestions = model.fetchPopQuiz(_rawContent);
+
+    return IbPopQuiz(
+      context: context,

```

Comment:
Why do we need to pass the context here?
For `Theme.of(context)` we don't need imo, please verify.

Also, check for `context` passing in other places too.

Reply:
Will check but since this class doesn't extends Widget, context will not be available.
Still let me check.

Reply:
Context is not available in a Stateless Widget, that's why we had to pass it.

Reply:
But I found that I was using context without any reasons in other builders 😂